setup.py

```diff
@@ -17,6 +17,12 @@
 import sys
 import logging
 
+if '--require-scandir-extension' in sys.argv:
+    sys.argv = [arg for arg in sys.argv if arg != '--require-scandir-extension']
+    require_scandir_extension = True
+else:
+    require_scandir_extension = False
+
```

Comment:
Did the environment variable approach not work? (If not, why not?) I'd much prefer an environment variable than messing with `sys.argv` in this way. Let's use `require_c_extension = os.getenv('SCANDIR_REQUIRE_C_EXTENSION') == '1'`.

Additionally, the use of `pip install --global-option` has a [big warning on it in the docs](https://pip.pypa.io/en/stable/reference/build-system/setup-py/#customising-the-build).

Reply:
done.

was not sure a bit, so I checked that https://github.com/benhoyt/scandir/pull/145 fails.